docs/data_node/extract_node/tube.md

```diff
@@ -0,0 +1,70 @@
+---
+title: TubeMQ
+sidebar_position: 11
+---
+
+import {siteVariables} from '../../version';
```

Comment:
![image](https://user-images.githubusercontent.com/107393625/189800884-ccf0c915-f94a-4323-ae97-d253278a20e1.png)

Reply:
what does it means?